### DISCUSSION TITLE
Add local IP info - IP, DNS server, Gateway

### DISCUSSION THREAD

**Discussion by jcolinger3** [InternetConnectionMonitor/icm-public#119] [Ideas and Feature requests]
Not sure if it is possible, but I feel it would be very helpful to have that local IP info available as well.
Local IP, DNS server(s), Gateway.
While it is simple for a tech person to check this, sometimes the end users are sure how to check this... but it is critical to troubleshooting an internet connection.

**Comment by dmishh** [comment]
@jcolinger3 Thanks for the idea. However, this information is unavailable due to browser limitations. Even retrieving a local IP requires some extra manipulations with Chrome flags: https://stackoverflow.com/questions/20194722/can-you-get-a-users-local-lan-ip-address-via-javascript

I'll think of adding some instructions that will allow to have all the information about troubleshooting the internet connection consolidated within the extension.
